shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Examples/Consensus.hs

```diff
@@ -127,7 +128,11 @@ defaultShelleyLedgerExamples mkWitnesses mkValidatedTx value txBody auxData =
             Right (KeyHashObj (mkKeyHash 2))
           ],
       sleResultExamples = resultExamples,
-      sleNewEpochState = exampleNewEpochState value,
```

Comment:
@JaredCorduan I'm glad to see you're criticizing this definition too.

Are there any ways you'd like to enrich its contents? I think that would be welcome, as long as it was in at least a separate commit so that we could validate the necessary golden-expectations updates.

Reply:
The existing example code seemed to be written for use in multiple eras, so I tried to get as much re-use out of it as I could.

The existing examples (Shelley, Allegra, and Mary) have not been changed here, but I can use `exampleNewEpochState` with different protocol parameters in Alonzo.

Reply:
Just to be clear, I think you were suggesting I use a separate commit because of potential gold tests breaking. But since no golden tests have actually been broken, I think you are probably happy with one commit.